cmd/root.go

```diff
@@ -78,6 +78,8 @@ type Command struct {
 	telemetryPrefix            string
 	prometheus                 bool
 	prometheusNamespace        string
+	prometheusAddress          string
```

Comment:
We also need httpAddress, right? Otherwise, the probe will fail (am I missing something?).